docs/pages/distribution/publishing-websites.mdx

```diff
@@ -26,153 +43,246 @@ If you make changes to your project, you'll need to rebuild it for production. D
 
-<Step label="4">
+```sh public/_redirects
+/*    /index.html   200
+```
 
-The Amplify Console will detect that the **amplify.yml** file is in your repo. Choose **Next**.
+If you modify this file, you will need to rebuild your project with `npx expo export -p web` to have it safely copied into the **dist** directory.
 
-</Step>
+</Tab>
 
-<Step label="5">
+<Tab label="webpack">
 
-Review your settings and choose **Save and deploy**. Your app will now be deployed to a `https://branchname.xxxxxx.amplifyapp.com` URL.
-
-</Step>
+If your app implements any navigation, you'll need to configure Netlify to redirect requests to the single **web-build/index.html** file. This can be done in Netlify by creating a `./public/_redirects` file and redirecting all requests to `/index.html`.
 
-### Vercel
+Navigate inside the **web-build** directory and run the following command to create **\_redirects** file with rule:
 
-[Vercel](https://vercel.com/) has a single-command zero-config deployment flow.
+```sh web/_redirects
+/*    /index.html   200
+```
 
-> For more information on unlimited hosting, check out [Vercel's pricing](https://vercel.com/pricing).
+If you modify this file, you will need to rebuild your project with `npx expo export:web` to have it safely copied into the **web-build** directory.
 
-<Step label="1">
+</Tab>
 
-[Install the Vercel CLI](https://vercel.com/download).
+</Tabs>
 
 </Step>
 
-<Step label="2">
+<Step label="3">
 
-Build your Expo web app with the `npx expo export:web` command.
+Deploy the web build folder by running the following command:
 
-</Step>
+<Tabs>
+  <Tab label="Expo Router">
+    <Terminal cmd={['$ netlify deploy --dir dist']} />
+  </Tab>
 
-<Step label="3">
+  <Tab label="webpack">
+    <Terminal cmd={['$ netlify deploy --dir web-build']} />
+  </Tab>
+</Tabs>
 
-To deploy, navigate to the **web-build** directory and run the `vercel` command:
+You will see a URL that you can use to view your project online.
 
-<Terminal cmd={['$ cd web-build', '', '$ vercel']} cmdCopy="cd web-build && vercel" />
+</Step>
 
-You should now see a URL that you can use to view your project online. Paste that URL into your browser when the build is complete, and you will see your deployed app.
+#### Continuous delivery
 
-</Step>
+Netlify can also build and deploy when you push to git or open a new pull request:
 
-### Surge
+- [Start a new Netlify project](https://app.netlify.com/signup).
+- Pick your Git hosting service and select your repository.
+- Click **Build your site**.
 
-[Surge](https://surge.sh/) offers static web publishing directly from a command line interface.
+### Vercel
 
-<Step label="1">
+[Vercel](https://vercel.com/) has a single-command deployment flow.
 
-[Install the surge CLI.](https://surge.sh/help/getting-started-with-surge)
+> For more information on hosting, check out [Vercel's pricing](https://vercel.com/pricing).
```

Comment:
I think we can remove the link to their pricing page. It's common knowledge that most services we've listed will here have paid plans

Reply:
```suggestion
```